autoria.py

```diff
@@ -0,0 +1,242 @@
+    for the given example it would be 1.
+
+    Args:
+        item_to_select - string, could be not 100% accurate as it is in the
+            auto.ria.ua lists, e.g. value ''Харьков'' is acceptable,
+            because the parameter is wildcarted in comprasion like
+            ''*Харьков*'', the function will find suitable name
+            ''Харьковская'' and will return its id.
+        items_list - JSON-formatted list of pairs ''name: value'',
+                obtained from one of the ''get_'' functions.
+
+    Returns:
+        integer with needed item (category, bodystyle, mark etc.) identifyer.
+    """
+    if item_to_select is not None and items_list is not None:
+        for item in items_list:
+            if fnmatch(item['name'], '*' + item_to_select + '*'):
+                return item['value']
+    else:
+        return None
+
+
+def get_average(category: str, mark: str, model: str, bodystyle: str = None,
+                years_start_end: list = None, state: str = None) -> json:
+    """Compose parameters for GET request and send it.
+
+    Get average price using search parameters.
+
+    Args:
+        category - (required) string with vehicle type,
+                   e.g. ''Легковые''
+        mark - (required) string with mark, like ''Renault''
+        model - (requred) sting with model, like ''Scenic''
+        bodystyle - (optional) string with bodystyle, e.g. ''Седан''
+        year_start_end - (optional) list with start and end mfr year,
+                        e.g. ''[2005, 2006]'', also one of years could be
+                        ''None'', e.g. ''[2005, None]''
+            <== note: make this parameter behave like on auto.ria: if there's
+            only start year end=current_year, vice versa - start=1950==>
+        state - (optional) string with state (city), e.g. ''Харьков''
+        ******************************************************************
+        The following search parameters are not in use right now.
+        Are to be added.
+        gear_type - (optional) list with gearshift types,
+                    e.g. ''[Механическая, Автоматическая]''
+        fuel - (optional) list with gasoline types,
+                e.g. ''[Бензин, Газ/Бензин]''
+        drive - (optional) list with drive types,
+                e.g. ''[Передний, Полный]''
+        'engine': 'engineVolume',
+        'options': 'options',
+        'mileage': 'raceInt',
+        'doors': 'door',
+        'state': 'state_id',
+        'city': 'city_id',
+        'carrying': 'carrying',
+        'seats': 'seats',
+        'color': 'color_id',
+        'custom': 'custom',
+        'damage': 'damage',
+        'credit': 'under_credit',
+        'confiscated': 'confiscated_car',
+        'on_repairment': 'onRepairParts',
+        ******************************************************************
+
+        Returns:
+            JSON-formatted output with data as described here:
+            https://goo.gl/WVCVi8
```

Comment:
The link is good - keep it, but the example is better, we can shorten it and it will still be clear enough:

```
{
    total: 17,
    arithmeticMean: 16305.882352941177,
    interQuartileMean: 8483.333333333334,
    percentiles: {
        1.0: 1944,
        5.0: 2520,
        25.0: 3500,
        ...
        99.0: 64868
    },
    prices: [
        67700,
        27000,
        ...
    ],
    classifieds: [
        14663610,
        14226353,
        ...
    ]
}
```